src/test/regress/sql/shard_rebalancer.sql

```diff
@@ -902,17 +930,23 @@ WHERE table_schema = 'public'
 ) a ORDER BY table_name;
 
 \c - - - :master_port
-SELECT * FROM get_rebalance_table_shards_plan('tab', rebalance_strategy := 'by_disk_size');
+SELECT * FROM get_rebalance_table_shards_plan('tab', rebalance_strategy := 'by_disk_size') ORDER BY shardid;
```

Comment:
Needs a comment on why this needs order by. I guess in this case it's because the disk sizes of the shard groups are normally the same, but sometimes randomly one is slightly bigger than the other, resulting in a different shard group being moved first.